torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -644,7 +644,11 @@ def _affine_bounding_box_xyxy(
         new_width, new_height = _compute_affine_output_size(affine_vector, width, height)
         spatial_size = (new_height, new_width)
 
-    return out_bboxes.to(bounding_box.dtype), spatial_size
+    out_bboxes = clamp_bounding_box(
```

Comment:
Shouldn't `clamp_bounding_box()` accept a BoundingBox instead of a Tensor? In other word should it be at the `dispatcher kernel` level instead of being a low-level kernel? Could we make it *both*?

Reply:
Fixed in #7227.